### PR TITLE
refactor polygon_backfiller to use mkts.yml

### PR DESCRIPTION
**Pull request by briancappello**
This PR refactors the `polygon_backfiller` command to load settings from the user's configured `mkts.yml` file.

Settings are loaded for the `ondiskagg` trigger, as well as all of the polygon-specific settings (api key, symbols, bars/quotes/trades, query_start). If symbols is left unspecified, fetch all the things as before. (but, maybe it'd be better to only fetch all tradable-on-Alpaca symbols?)

Also, it may be preferable to instead make this a Cobra command? (eg `marketstore backfill ...`)

### CODE REVIEW

File: contrib/polygon/backfill/backfiller/backfiller.go [hunk 1]
```diff
@@ -74,79 +63,59 @@ func main() {
...
+		log.Info("[polygon] fetching available symbols")
+		resp, err := api.ListTickers()
+		if err != nil {
+			log.Fatal("[polygon] failed to list symbols (%v)", err)
+		}
+
+		symbolList := make([]string, 1)
+		for _, s := range resp.Tickers {
 			symbolList = append(symbolList, s.Ticker)
 		}
```
Comment (dakimura) [inline contrib/polygon/backfill/backfiller/backfiller.go:79]: [nit] slightly faster as there will be no new allocation
```suggestion
		symbolList := make([]string, len(resp.Tickers))
		for i, s := range resp.Tickers {
			symbolList[i] = s.Ticker
		}
```

Comment (briancappello) [reply contrib/polygon/backfill/backfiller/backfiller.go]: Ah nice, thanks!

**Comment by eli-kaplan** [comment]
Just tested this out - there seems to be a typo in the `symbolList` pre-allocation (`contrib/polygon/backfill/backfiller/backfiller.go`, line 76):

The current code:
`symbolList := make([]string, len(resp.Tickers))` 
is shadowing `symbolList` within that block, so once it leaves that block, `symbolList` is empty again. Hence, after retrieving all 12,000 or so symbols from Polygon, the backfiller 'selects' zero symbols and just exits.

It should instead be:
`symbolList = make([]string, len(resp.Tickers))`
which results in the expected behavior.

**Comment by briancappello** [comment]
@eli-kaplan Thanks for catching that, fixed!

**Review by dakimura — APPROVED** [review]

**Comment by tibkiss** [comment]
@briancappello : Thank you for this contribution. 

I agree that it is a better approach to load certain configurations from mkts.yaml config, however we need
the ability to specify (override) some of the parameters from command line.

I took a quick glimpse and it appears to me that exchange filtering functionality was removed in this change,
which is also required for our operations.

I'll allocate some time and go through on this more precisely. 

Until then please do not merge as it will break our operations. Thanks.

CC: @umitanuki / @leki75

**Comment by briancappello** [comment]
Superseded by https://github.com/alpacahq/marketstore/pull/380

**Comment by tibkiss** [comment]
@briancappello : I believe this PR is a very nice addition to the backfilling capabilities. 
Unfortunately due to other merged pr (#380) this now cannot be merged due to conflict.

If you don't mind I'd like to contribute here by resolving the conflict and adding back the exchange
filter capability. WDYT?

**Comment by tibkiss** [comment]
I have resolved the conflicts and made some adjustments here:
https://github.com/alpacahq/marketstore/pull/394

**Pull request closed without merging**